recipes-domd/domd-image-weston/domd-image-weston.bbappend

```diff
@@ -69,3 +69,10 @@ configure_versions_rcar() {
 python do_configure_append_rcar() {
     bb.build.exec_func("configure_versions_rcar", d)
 }
+
+do_install_append () {
+    local LAYERDIR=${TOPDIR}/../meta-xt-prod-gen3-test
+    find ${LAYERDIR}/doc -iname "u-boot-env*" -exec cp -f {} ${DEPLOY_DIR}/domd-image-weston/images/${MACHINE}-xt \; || true
+    find ${LAYERDIR}/doc -iname "mk_sdcard_image.sh" -exec cp -f {} ${DEPLOY_DIR}/domd-image-weston/images/${MACHINE}-xt \; || true
```

Comment:
I was thinking where to store this script. Taking into the account that we feed this script with the root images directory, would it be better to put script there?